docs/source/upcoming_release_notes/1189-flow-switch.rst

```diff
@@ -0,0 +1,32 @@
+1189 flow-switch
+#################
+
+API Breaks
+----------
+- N/A
+
+Features
+--------
+- pcdsdevices now has a `digital_signals.py` module for simple digital io.
```

Comment:
Not 100% sure but I think this also makes a sphinx auto link without the .py
```suggestion
- pcdsdevices now has a `digital_signals` module for simple digital io.
```

Reply:
sphinxed